README.md

```diff
@@ -202,12 +202,6 @@ You can deploy this project using:
 - GitHub: [@ijosue16](https://github.com/ijosue16)
 
 
```

Comment:
- [ ] You have duplicate **Built With** sections on your readme file. Kindly fix that.

- [ ] Kindly include the key features of the application

  ![fhfd](https://user-images.githubusercontent.com/81468675/207271801-66dd3610-7fe1-43b3-b8bc-3c6267c585f5.jpg)

Reply:
please check the README.md

Reply:
@ijosue16 
Your project is complete! 
Cheers! 😃